helix-term/src/commands/typed.rs

```diff
@@ -1734,13 +1734,30 @@ fn insert_output(
     Ok(())
 }
 
+fn pipe_to(
+    cx: &mut compositor::Context,
+    args: &[Cow<str>],
+    event: PromptEvent,
+) -> anyhow::Result<()> {
+    _pipe(cx, args, event, &ShellBehavior::Ignore)
+}
+
 fn pipe(cx: &mut compositor::Context, args: &[Cow<str>], event: PromptEvent) -> anyhow::Result<()> {
+    _pipe(cx, args, event, &ShellBehavior::Replace)
+}
+
+fn _pipe(
```

Comment:
```suggestion
fn pipe_impl(
```

is the common pattern in the codebase